app_dev.yaml

```diff
@@ -113,6 +113,18 @@ handlers:
   secure: always
   expiration: "0"
 
+# PRODUCTION/DEVELOPMENT DYNAMIC
+# Note: Redirection is disabled (using 'secure: never') for requests that start
```

Comment:
How did this work in Python 2?

Reply:
Like this https://github.com/oppia/oppia/blob/c89776f4857746a0f7a23e09c1ceb445f6d4bdde/app_dev.yaml#L135-L138, I think because the `login` was allowed it worked, but now it needs `secure: never`.